### PR TITLE
Novacustom + MSI changes

### PR DESCRIPTION
**Pull request by PLangowski**

### CODE REVIEW

File: docs/unified/msi/firmware-transition.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+!!! warning "Warning for Qubes OS users"
+
+    If you installed Qubes OS under Heads, it has been installed in legacy BIOS
+    boot mode and will not be bootable under UEFI. You will likely need to
+    install Qubes OS again.
+
+    Users who installed Qubes OS *before* switching to Heads are not affected.
+
+## Logo customization
```
Comment (artur-rs) [inline docs/unified/msi/firmware-transition.md:45]: @PLangowski same question as in the novacustom section

File: docs/unified/novacustom/building-manual.md [hunk 2]
```diff
@@ -1,228 +1,280 @@
...
 
 ## Install Dasharo firmware
 
 The Dasharo firmware can be flashed in following ways, depending on your
 situation:
 
-- To flash Dasharo for the first time, refer to the
-  [initial deployment guide](initial-deployment.md).
-- To update Dasharo, refer to the [firmware update guide](firmware-update.md).
+    - To flash Dasharo for the first time, refer to the
```
[the remark this answers is not in the document]
Comment (PLangowski) [reply docs/unified/novacustom/building-manual.md]: Fixed 5ae6b425b31e99e6783b4b57480367c7777101cf

File: docs/unified/msi/firmware-transition.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+Dasharo Entry Subscription or not running Dasharo v1.1.3) and then use
+FlashBIOS with heads binary.
+
+To transition to heads firmware with flashrom, whole binary has to be flashed:
+
+```bash
+flashrom -p internal -w <heads_fw_file>
+```
+
+!!! warning "ME has to be disabled to flash full file."
```
[the remark this answers is not in the document]
Comment (PLangowski) [reply docs/unified/msi/firmware-transition.md]: My bad 5ae6b425b31e99e6783b4b57480367c7777101cf

File: docs/unified/novacustom/firmware-transition.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+Once finished, your laptop will shut down automatically. Power it back on to
+boot into your UEFI firmware.
+
+> When reverting to UEFI, it's not possible to restore EFI boot manager entries
+> that were added before installing Heads. Therefore, you may need to re-create
+> your boot entries manually, or find your boot loader using `Boot From File`
+> option in the UEFI setup menu.
+
+## Logo customization
```
[the remark this answers is not in the document]
Comment (artur-rs) [reply docs/unified/novacustom/firmware-transition.md:78]: @PLangowski, it should be moved to a more generic place, like `Initial deployment`, or even added as a new entry in the `FAQ` section with the link to the `Initial deployment/Logo customization` section.

File: docs/unified/msi/firmware-transition.md (hunk 1 above)
Comment (artur-rs) [reply docs/unified/msi/firmware-transition.md:50]: https://github.com/Dasharo/docs/pull/804#discussion_r1615931078

File: docs/unified/novacustom/overview.md [hunk 5]
```diff
@@ -15,8 +15,8 @@ Select your Dasharo firmware flavor:
 
         <div class="grid cards" markdown>
 
-        - [NS51 Series](https://configurelaptop.eu/ns51-series/)
-        - [NS70 Series](https://configurelaptop.eu/ns70-series/)
+        - [NS51 Series](https://configurelaptop.eu/product/ns51-series/)
```
[the remark this answers is not in the document]
Comment (PLangowski) [reply docs/unified/novacustom/overview.md]: Fixed 6b1b3ee5cfb1db0effffee60ab58a9cf1cb6d6b3

File: docs/unified/msi/firmware-transition.md (hunk 1 above)
Comment (PLangowski) [reply docs/unified/msi/firmware-transition.md]: 8836f189347c8858706b484326b63d4a4bad48b2

**Review by artur-rs — APPROVED** [review]

**Pull request merged**